src/test/java/edu/ksu/canvas/attendance/controller/StudentSummaryControllerITest.java

```diff
@@ -100,15 +100,13 @@ public void additionalSetup() throws NoLtiSessionException, ParseException {
         when(mockLtiSession.getCanvasCourseId()).thenReturn(String.valueOf(existingCourse.getCanvasCourseId()));
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void studentSummary_NonNumberSectionId() throws Throwable {
         Long studentId = existingStudent.getStudentId();
         String nonNumberSectionId = "Numbers";
-        try {
-            mockMvc.perform(get("/studentSummary/" + nonNumberSectionId + "/" + studentId));
-        } catch (NestedServletException ne) {
-            throw ne.getCause();
-        }
+        mockMvc.perform(get("/studentSummary/" + nonNumberSectionId + "/" + studentId))
+            .andExpect(view().name("forward:roster"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "forward:roster" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)